### PR TITLE
Created new note UI

### LINKED ISSUE #4: Create new note UI

### PR DESCRIPTION
**Pull request by KrishnapriyaSkk**
Changes:
1. Created new note UI.
2.  Added new note from validation.
3.  Addresses comments from previous PR review.

Fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 1]
```diff
@@ -30,3 +25,76 @@ export const USER_AVATAR_URL = "https://picsum.photos/id/1005/5760/3840";
...
+  },
+  {
+    id: 4,
+    name: "Bugs",
+  },
+  {
+    id: 5,
+    name: "V2",
+  },
+];
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js]: https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663335627615309?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

**Review by yedhink — CHANGES_REQUESTED** [review]
@KrishnapriyaSkk Go ahead and merge this PR once the following change is added.

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 1 above)
Comment (KrishnapriyaSkk) [reply app/javascript/src/components/Dashboard/Notes/constants.js]: The above changes were made. Thank you.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 2]
```diff
@@ -26,51 +28,21 @@ export const SIXTY = 60;
...
-  {
-    id: 2,
-    name: "Ronald Jones",
-  },
-  {
-    id: 3,
-    name: "Jacob Richards",
-  },
-];
+export const CONTACT_DETAILS_DATA = buildSelectOption([
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:31]: You are not building a single option here. You are building options, plural. Thus rename the util method appropriately.

**Review by yedhink — COMMENTED** [review]
@KrishnapriyaSkk

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 2 above)
Comment (KrishnapriyaSkk) [reply app/javascript/src/components/Dashboard/Notes/constants.js:31]: Okay. Thank you. Will make the appropriate changes.

**Pull request merged**